pullrequest.rst

```diff
@@ -258,21 +258,30 @@ and for each pull request there may be several commits.  In particular:
 
 Commit messages should follow the following structure::
 
-   bpo-42: the spam module is now more spammy (GH-NNNN)
+   bpo-42: make the spam module more spammy (GH-NNNN)
```

Comment:
This comment is based on my preference and looking at two pages of current  `git log --oneline` output.
```suggestion
   bpo-42: Make the spam module more spammy (GH-NNNN)
```
An initial capital is nearly always used if there is no bpo number.  The exception was of the form 's/type/fix/'.  When there is a bpo number, capitals are more common than not.
@gvanrossum Do you disagree with the change?

Reply:
@terryjreedy I agree with you. I saw the capital letter on various commit messages, too, and I'm using it as well. Furthermore, the devguide's [quick reference](https://devguide.python.org/#quick-reference) uses a capital letter in point seven, too.
If @gvanrossum agrees on this, I'm happy to apply the suggested change.

Reply:
I'm personally neutral about the change because I don't think it has a significant effect on the readability of the commit messages for the purposes of detailing the changes (unlike imperative vs descriptive, IMO). So while the prevailing style is sentence case capitalization after the `bpo-NNN:`, I don't know that it's a real concern.

That being said, if we just modified the example to use it (as proposed above) rather than writing about in the guidelines, I don't have an issue with it.